components/mixins/experience-event/events/removefromlist.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/events/removefromlist",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Remove From List",
+  "type": "object",
+  "meta:conditionalField": "xdm:eventType",
+  "meta:conditionalValue": "removefromlist",  
```

Comment:
This conditional value is what they pass to the "eventType" field. Please look at the existing list of potential values and use it, or propose a new value here. Use the same format if creating a new value - https://github.com/adobe/xdm/blob/master/components/behaviors/time-series.schema.json#L